app/main.ipynb

```diff
@@ -2,50 +2,269 @@
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>id</th>\n",
+       "      <th>datetime</th>\n",
+       "      <th>quantity</th>\n",
+       "      <th>order_id</th>\n",
+       "      <th>price</th>\n",
+       "      <th>name</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>6</td>\n",
+       "      <td>2019-08-03 20:25:00</td>\n",
+       "      <td>2</td>\n",
+       "      <td>1</td>\n",
+       "      <td>0.80</td>\n",
+       "      <td>Plain Papadum</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>240</td>\n",
+       "      <td>2019-08-03 20:25:00</td>\n",
+       "      <td>1</td>\n",
+       "      <td>1</td>\n",
+       "      <td>12.95</td>\n",
+       "      <td>King Prawn Balti</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>19</td>\n",
+       "      <td>2019-08-03 20:25:00</td>\n",
+       "      <td>1</td>\n",
+       "      <td>1</td>\n",
+       "      <td>2.95</td>\n",
+       "      <td>Garlic Naan</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>36</td>\n",
+       "      <td>2019-08-03 20:25:00</td>\n",
+       "      <td>1</td>\n",
+       "      <td>1</td>\n",
+       "      <td>3.95</td>\n",
+       "      <td>Mushroom Rice</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>111</td>\n",
+       "      <td>2019-08-03 20:25:00</td>\n",
+       "      <td>1</td>\n",
+       "      <td>1</td>\n",
+       "      <td>8.95</td>\n",
+       "      <td>Paneer Tikka Masala</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    id             datetime  quantity  order_id  price                 name\n",
+       "0    6  2019-08-03 20:25:00         2         1   0.80        Plain Papadum\n",
+       "1  240  2019-08-03 20:25:00         1         1  12.95     King Prawn Balti\n",
+       "2   19  2019-08-03 20:25:00         1         1   2.95          Garlic Naan\n",
+       "3   36  2019-08-03 20:25:00         1         1   3.95        Mushroom Rice\n",
+       "4  111  2019-08-03 20:25:00         1         1   8.95  Paneer Tikka Masala"
+      ]
+     },
+     "execution_count": 56,
+     "metadata": {},
+     "output_type": "execute_result"
     }
-   }
+   ],
+   "source": [
+    "\n",
+    "# Save the results to a CSV file\n",
+    "df.to_csv(\"restataraunt_data.csv\", index=False)\n",
+    "df.head()"
```

Comment:
It is important to show here that your result pass the quick check from the task 0 description.
Also remove the id column. Change columns order so 'name' will be on the first place